src/content.ts

```diff
@@ -53,24 +53,12 @@ messaging.addListener("alive", async () => true)
 
 // Hook the keyboard up to the controller
 import * as ContentController from "@src/content/controller_content"
-import { getAllDocumentFrames } from "@src/lib/dom"
 
 const guardedAcceptKey = (keyevent: KeyboardEvent) => {
     if (!keyevent.isTrusted) return
     ContentController.acceptKey(keyevent)
 }
 function listen(elem) {
-    elem.removeEventListener("keydown", guardedAcceptKey, true)
-    elem.removeEventListener(
-        "keypress",
-        ContentController.canceller.cancelKeyPress,
-        true,
-    )
-    elem.removeEventListener(
-        "keyup",
-        ContentController.canceller.cancelKeyUp,
-        true,
-    )
```

Comment:
These are added again below, in the same exact form, so removing them first is not necessary.